mozilla_schema_generator/configs/deletion_request.yaml

```diff
@@ -1,13 +1,12 @@
 payload:
-  processes:
+  scalars:
     parent:
-      scalars:
-        match:
-          table_group: scalars
-          type: scalar
-          details:
-            keyed: false
-            record_in_processes:
-              contains: main
-            record_into_store:
-              contains: deletion-request
```

Comment:
Since the schema has already been deployed, should the old configuration be kept? Otherwise those fields will disappear.

Reply:
Yes, I believe it will be necessary to keep the old fields.